group-generators/generators/ethereum-developers/index.ts

```diff
@@ -0,0 +1,35 @@
+import { Group, Tags, ValueType } from "../../../src/topics/group";
+import {
+  GenerationFrequency,
+  GroupGenerator,
+} from "../../../src/topics/group-generator";
+import { GenerationContext } from "../../../src/topics/generation-context";
+import BigQueryProvider from "../../helpers/providers/big-query/big-query";
+
+/*
+  Ethereum Developers group is constituted of Ethereum accounts that deployed at least one contract
```

Comment:
Shouldn't we add a description field inside `Group` ? It would be nice to get this from the api ...

Reply:
Mmm need to think about it :-)